### ISSUE TITLE
Change from MIT license style to Open Source.

### ISSUE THREAD

**Issue by croeber** [ngx-rocket/starter-kit#15]
I don't want to have to have my software a MIT license type just because it is derived from a template that is MIT license type. I love your template though, good work.

**Comment by sinedied** [comment]
@croeber  I think you're mistaking MIT license for GPL and open-source:
- The MIT license IS open-source, and one of the most permissive one, see https://choosealicense.com/licenses/mit/
- Only the starter template is under MIT license: you can license your own added and derived code under your own license, even make it proprietary if you like (it's NOT like the GPL license that contaminate derived work, see https://choosealicense.com/licenses/gpl-3.0/). The only limitation is that you cannot claim you made this starter kit, and you cannot make us liable for any issues regarding this starter kit.
- If you want to start with code free of any license, do not use this starter kit but use the [generator](https://github.com/ngx-rocket/generator-ngx-rocket) instead: it will generate the same code (among many other possibilities), but generated code is license-free.

TL;DR: the MIT license is essentially a legal protection for us, and the only restriction is to keep the original license file somewhere in your project, other than that you can do whatever you want with it.
